App.js

```diff
@@ -0,0 +1,39 @@
+const express = require("express");
+const bodyParser = require("body-parser");
+const cors = require("cors");
+const app = express();
+const db = require("./models");
+const controller = require("./controllers/All.controller.js");
+
+var corsOptions = {
+    origin: "http://localhost:8081"
+};
+app.use(cors(corsOptions));
+// parse requests of content-type - application/json
+app.use(express.json());
+// parse requests of content-type - application/x-www-form-urlencoded
+app.use(express.urlencoded({ extended: true }));
+// simple route
+app.get("/", (req, res) => {
+    res.json({ message: "Welcome to bezkoder application." });
+});
+// db.sequelize.sync({ force: true }).then(() => {
+//     console.log( "Drop and re-sync db.");
+// }); 
+
+
+app.post("/CreateUser", controller.createUser);
```

Comment:
`post` is self explanatory that we are creating user.  The end point just be `users`